packages/core/plugin.schema.json

```diff
@@ -7,15 +7,15 @@
           "properties": {
             "didDocument": {
               "$ref": "#/components/schemas/DIDDocument",
-              "description": "the DID document from which to extract the fragment. This MUST be the document resolved by  {@link  resolveDid  }"
+              "description": "the DID document from which to extract the fragment. This MUST be the document resolved by\n {@link  IResolver.resolveDid }"
             },
             "didUrl": {
               "type": "string",
-              "description": "The DID URI that refers to the subsection by #fragment. Example: did:example:identifier#controller"
+              "description": "The DID URI that needs to be dereferenced. This should refer to the subsection by #fragment.\n\nExample: did:example:identifier#controller"
             },
             "section": {
               "$ref": "#/components/schemas/DIDDocumentSection",
-              "description": "The section of the DID document where to search for the fragment. Example 'keyAgreement', or 'assertionMethod', or 'authentication', etc"
+              "description": "Optional. The section of the DID document where to search for the fragment. Example 'keyAgreement', or 'assertionMethod', or 'authentication', etc"
```

Comment:
I agree, however these comments serve multiple purposes: schema, IDE tooltips, and reference documentation on the website. I believe the redundancy is not too much here, especially since these plugin schemas are automatically generated.